FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies.rdf

```diff
@@ -1909,5 +1909,14 @@ The ABA RTN is necessary for the Federal Reserve Banks to process Fedwire funds
 		<rdfs:range rdf:resource="&fibo-fbc-fct-usjrga;PrimaryFederalRegulator"/>
 		<skos:definition>identifies the primary federal regulator for an institution</skos:definition>
 	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fbc-fct-usjrga;hasSecondaryFederalRegulator">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-fct-rga;isRegulatedBy"/>
+		<rdfs:label>has secondary federal regulator</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-pas-fpas;FinancialServiceProvider"/>
+		<rdfs:isDefinedBy rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies/"/>
+		<rdfs:range rdf:resource="&fibo-fbc-fct-rga;RegulatoryAgency"/>
+		<skos:definition>identifies the secondary federal regulator for an institution</skos:definition>
```

Comment:
Sorry.  I'll have to do some research about what this means.  But not in this issue - please raise an issue for that.

Reply:
Hard to raise an issue on something that's not been added yet (until this is merged). What's the urgency for adding this to a Release ontology - can we not hold off until it's better understood and defined? I don't see that it's needed for this SEC issue.

Reply:
Not all institutions have secondary regulators.  Wells Fargo does, primarily because of some of the issues raised with respect to business practices over the last few years, but it is not obvious for the other institutions.  It is likely that they all have multiple regulators, but perhaps they are not all as visibly identified.  In any case, I did not feel comfortable adding a restriction, but revised the definition so that there could be one primary and multiple secondary regulators.